baselines/ppo2/runner.py

```diff
@@ -166,6 +194,32 @@ def other_agent_action():
         mb_returns = mb_advs + mb_values
         return (*map(sf01, (mb_obs, mb_returns, mb_dones, mb_actions, mb_values, mb_neglogpacs)),
             mb_states, epinfos)
+
+    def randomly_set_tom_params(self):
+        """
+        Randomly choose which TOM params to use for each agent in the parallel envs
+        """
```

Comment:
This function also seems like it belongs in your project repo, maybe as a method of the TOM agent?